.travis.yml

```diff
@@ -1,7 +1,9 @@
 language: node_js
 sudo: false
+distro: bionic
```

Comment:
`distro` -> `dist`? https://docs.travis-ci.com/user/reference/overview/

Reply:
Looks like both `dist` and `distro` work https://travis-ci.com/github/nextstrain/auspice/jobs/307552818#L9 , but good to know